rfcs/InputUnion.md

```diff
@@ -160,9 +162,48 @@ If a solution places any restrictions on member types, compliance with these res
 
 In addition to containing Input types, member type may also contain Leaf types like `Scalar`s or `Enum`s.
 
-* Objection: Multiple Leaf types serialize the same way, making it impossible to distinguish the type. For example, a `String`, `ID` and `Enum`.
+* Objection: Multiple Leaf types serialize the same way, making it impossible to distinguish the type without additional information. For example, a `String`, `ID` and `Enum`.
+  * Potential solution: only allow a single built-in leaf type per input union.
 * Objection: Output polymorphism is restricted to Object types only. Supporting Leaf types in Input polymorphism would create a new inconsistency.
 
+### Changing field from an input type to an input union including that type is non-breaking
+
+Since the input object type is now a member of the input union, existing input objects being sent through should remain valid.
+
+* Objection: achieving this by indicating the default in the union (either explicitly or implicitly via the order) is undesirable as it may require multiple equivalent unions being created where only the default differs.
+* Objection: achieving this by indicating a default type in the input field is verbose/potentially ugly.
+
+### Input unions may include other input unions
```

Comment:
Not sure I understand this one - what does it mean to "include" another input union?

Reply:
```
inputUnion U1 = A | B
imputUnion U2 = U1 | C
```

Here U2 includes U1 (it does not need to explicitly list `A` and `B` as members, making maintenance easier).